src/modules/listComments..js

```diff
@@ -0,0 +1,22 @@
+const listComment = (id) => {
+  const commentsSection = document.getElementById('comments-section');
+  let scoreArray = [];
+  const COMMENTLLIST_URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/eeE9kmiK9ON5LxnmgZ0i/comments?item_id=';
+  const addToList = async () => {
+    const result = await fetch(`${COMMENTLLIST_URL}${id}`).then((res) => res.json());
+    return result;
+  };
+  addToList().then((res) => {
+    if (res) {
```

Comment:
Please consider starting the `addToList` function by emptying the list section first before populating it. This will ensure the list doesn't grow longer with duplicate content. You can achieve this using the code below on line 10
```JavaScript 
commentsSection.innerHTML = '' ;
```